tests/repository_data/targets_simpler.json

```diff
@@ -0,0 +1,54 @@
+{
+ "signatures": [
+ ],
+ "signed": {
+  "_type": "Targets",
+  "delegations": {
+   "keys": {
+    "a0650f29dde63403cc4eec28a1c66f2262d6339434a01c63a881a48bedd9bca9": {
+     "keyid_hash_algorithms": [
+      "sha256",
+      "sha512"
+     ],
+     "keytype": "ed25519",
+     "keyval": {
+      "public": "3b11296fe2dba14a2ef204e542e9e4195293bcf3042655e3d7e4ef5afe3cf36a"
+     }
+    }
+   },
+   "roles": [
+    {
+     "backtrack": true,
```

Comment:
That's true. One of sadly many things that will have to be adjusted when the merge happens.